corehq/apps/hqwebapp/static/cloudcare/scss/font-formplayer.scss

```diff
@@ -0,0 +1,6 @@
+// Based on the Structure of FontAwesome
+
+@import "font-formplayer/variables.less";
```

Comment:
curious how these `.less` files are being handled by `.scss`? Maybe the extension needs to change?

Also,
```
@import "font-formplayer/variables";
```
should be enough yes?

Reply:
I assume sass would break if they were used anywhere, as they're full of syntax errors - like using less-style`@variable` instead of sass-style `$variable`. But they're inert right now.